src/renderers/shared/fiber/ReactDebugCurrentFiber.js

```diff
@@ -49,10 +51,23 @@ function getCurrentFiberStackAddendum(): string | null {
   return null;
 }
 
+function resetCurrentFiber() {
+  ReactDebugCurrentFrame.getCurrentStack = null;
```

Comment:
I realized that this is not handling reentrancy. This means that we can lose stack information once we call another React. Such as in [this unit test](https://github.com/facebook/react/blob/master/src/renderers/dom/shared/__tests__/ReactServerRendering-test.js#L540).